src/ui/components/Button/Button.module.css

```
@@ -20,6 +20,29 @@
+
+.secondary {
+  background-color: transparent;
+  border: 2px solid var(--color-brand);
+  color: var(--color-brand);
+}
+
+.spinner {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  border: 2px solid currentColor;
```

Stylelint (17.2.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)